doc/api/esm.md

```diff
@@ -341,15 +341,18 @@ will be used as the final fallback.
 
 The conditions supported in Node.js are matched in the following order:
 
-1. `"require"` - matched when the package is loaded via `require()`.
+1. `"node"` - matched for any Node.js environment. Can be a CommonJS or ES
+   module file. _This is currently only supported behind the
+  `--experimental-conditional-exports` flag._
+2. `"require"` - matched when the package is loaded with the CommonJS
+  module resolver such as via `require()`.
```

Comment:
I think talking about resolvers is too technical for the docs.

```md
2. `"require"` - matched when the package is loaded via `require()`.
```

Reply:
Sure, as long as we in the group remain aware of the difference and don't start thinking these are the true definitions.

Reply:
I think we will. If you want _that_ detail somewhere in the docs, maybe down in the section where you define the resolver spec? That feels more like the place for it.